salt/modules/oracle.py

```diff
@@ -1,228 +1,687 @@
 # -*- coding: utf-8 -*-
-'''
-Oracle DataBase connection module
-
-:maintainer: Vladimir Bormotov <bormotov@gmail.com>
-
-:maturity: new
-
-:depends: cx_Oracle
-
-:platform: all
-
-:configuration: module provide connections for multiple Oracle DB instances.
-
-    **OS Environment**
+#
+# The MIT License (MIT)
```

Comment:
@thatch45 I was :100:% sure you will say that! :smile:  
There are couple of reasons:
1. MIT is less restrictive than Apache 2.0
2. The code is actually mostly moved from our SMDBA project (https://github.com/suse/smdba) which has MIT licence. It would be better to keep it that way.
3. MIT licence is not really new in Salt, e.g.: https://github.com/saltstack/salt/blob/develop/salt/modules/xfs.py

But I can _promise_ not introducing anything else in a future! :grinning: 

Also I see there some lint issues, I will fix them tomorrow.